lib/src/package.dart

```diff
@@ -34,6 +34,8 @@ class Package {
   /// The path to the directory containing the package.
   final String dir;
 
+  bool get isInMemory => dir == null;
```

Comment:
This doesn't have to be public does it? Can we also give a single line doc-comment.